_posts/2022-04-04-inclusive-language-guide.md

```diff
@@ -0,0 +1,25 @@
+---
+category: blog-detail
+post-type: blog
+by: the Scala Center
+title: "The Inclusive Language Guide for the Scala community"
+---
+
+As Scala stewards, we should all be committed to providing a friendly, safe and welcoming environment for all, regardless of age, body size, disability, ethnicity, sex characteristics, gender identity and expression, level of experience, education, socio-economic status, nationality, personal appearance, race, religion, sexual identity and orientation, or other such characteristics.
```

Comment:
because Scala Steward is so well-known, I think finding a different word than "steward" here might be a little better, idk maybe "members of the Scala community" or, idk, "keepers of the Scala flame", no that's too colorful... hmm...

Reply:
(the "we" is a bit ambiguous as whether it means everybody reading the post, or is meant to refer more specifically to Scala Center personnel and/or others who are in prominent, public, visible positions)

Reply:
Good point. I used "members of the Scala community". Hopefully that reflects the intention well.